scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -601,8 +602,9 @@ SG_ DUMMY_AIR_SHUTDOWN : 0|1@1+ (1,0) [0|1] "" DCM
 BO_ 510 DIM_MOTOR_SHUTDOWN_ERRORS: 8 DIM
 SG_ DUMMY_MOTOR_SHUTDOWN : 0|1@1+ (1,0) [0|1] "" DCM
 
-BO_ 600 CAN_DEBUGGING_SIGNALS: 1 CAN_DBG
-SG_ ENABLE_CELL_BALANCING : 0|1@1+ (1,0) [0|1] "" BMS
+BO_ 600 CAN_DEBUGGING_SIGNALS: 1 DEBUG
```

Comment:
rename to CELL_BALANCING?

Reply:
I want this to be a general message containing signals that we can use to take control of the bus